catalog/resourcemanagement/resourcemanagement-usage-ui/src/main/webapp/css/index.css

```diff
@@ -10,6 +10,22 @@
  *
  **/
 
+table {
+  table-layout: fixed;
+  border-collapse: collapse;
+  width: 100%;
+}
+
+td {
+  overflow: hidden;
```

Comment:
If you add `text-overflow: ellipsis;` to the td style it'll give an ellipsis as a hint for when for the text overflows.